.github/workflows/nightly.yml

```diff
@@ -2,110 +2,411 @@ name: Nightly build
 
 on:
   schedule:
-    - cron: '0 0 * * *' # run at midnight UTC
+    - cron: "0 0 * * *" # run at midnight UTC
   workflow_dispatch:
 
 jobs:
   build-archetype:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Java SE 8
```

Comment:
Please restore this. We want to build using the lowest Java SE version. Alternatively, we can build with each Java SE version under test.